content/courses/testing.md

```diff
@@ -6,19 +6,22 @@ slug: "/courses/Testing/"
 category: "Automated Testing"
 tags:
   - "Jest"
-description: "Welcome to the Automated Testing course. Over the upcoming weeks you will dive into 
-frontend testing using Jest and eventually Enzyme. By the end of this track you will know how 
-to test user interfaces, as well as user interactions." 
-lessons: 
+description: "Welcome to the Automated Testing course. Over the upcoming weeks you will dive into
+frontend testing using Jest, and eventually Enzyme or React Testing Library. By the end of this track you will know how
+to test user interfaces, as well as user interactions."
```

Comment:
This sentence reads a little weirdly. Could we replace it with this?

> By the end of this course, you will learn how to test interactive React components, simulate interactions with them, and test how your components behave in response to these interactions.

Reply:
I think updating this is a good idea, but theres a lot of repeating the word 'interactive' and 'interactions'... How about this:
> By the end of this course, you will know how
to test React components, simulate interactions with them, and test how they behave in response to these interactions.

Reply:
Love it!